backend/dal/Services/UserOrganizationService.cs

```diff
@@ -0,0 +1,255 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using MapsterMapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities;
+using Pims.Dal.Entities.Models;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Security;
+
+namespace Pims.Dal.Services
+{
+    /// <summary>
+    /// OrganizationService class, provides a service layer to administrate organizations within the datasource.
+    /// </summary>
+    public class UserOrganizationService : BaseService<PimsOrganization>, IUserOrganizationService
+    {
+        #region Variables
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a OrganizationService, and initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="dbContext"></param>
+        /// <param name="user"></param>
+        /// <param name="service"></param>
+        /// <param name="logger"></param>
+        public UserOrganizationService(PimsContext dbContext, ClaimsPrincipal user, IPimsService service, ILogger<UserOrganizationService> logger, IMapper mapper) : base(dbContext, user, service, logger, mapper) { }
+        #endregion
+
+        #region Methods
+
+        /// <summary>
+        /// Get a page of organizations from the datasource.
+        /// </summary>
+        /// <param name="page"></param>
+        /// <param name="quantity"></param>
+        /// <returns></returns>
+        public Paged<PimsOrganization> Get(int page, int quantity)
+        {
+            return Get(new OrganizationFilter(page, quantity));
+        }
+
+        /// <summary>
+        /// Get a page of organizations from the datasource.
+        /// The filter will allow queries to search for the following property values; Name, Description, ParentId.
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <returns></returns>
+        public Paged<PimsOrganization> Get(OrganizationFilter filter = null)
+        {
+            this.User.ThrowIfNotAuthorized(Permissions.SystemAdmin);
+
+            var query = this.Context.PimsOrganizations.AsNoTracking();
+
+            if (filter != null)
+            {
+                if (filter.Page < 1) filter.Page = 1;
+                if (filter.Quantity < 1) filter.Quantity = 1;
+                if (filter.Quantity > 50) filter.Quantity = 50;
+                if (filter.Sort == null) filter.Sort = System.Array.Empty<string>();
```

Comment:
very nitty nit: inconsistent if style compared to the ones below as well as other ones not brackets

Reply:
This is a "legacy" service that I renamed to make way for the "proper" OrganizationService. I would be inclined to leave as-is because not part of this PR (other than the name change)